types/datatables.net/index.d.ts

```diff
@@ -704,7 +704,7 @@ declare namespace DataTables {
         /**
          * Get the footer th / td cell for the selected column.
          */
-        footer(): any;
+        footer(): Node;
```

Comment:
You seem to cast this often - if it really returns the footer th/td cell, doesn't that make it always an `HTMLElement`, and not just a `Node`?

Reply:
I don't disagree with you, but this fits the [documentation](https://datatables.net/reference/api/column().footer()), both in the JSDoc comments, as well as the function return type. Seeing the return type of `any` caught my attention, so I looked it up.

Reply:
The usage of [`node`](https://datatables.net/reference/type/node) in their docs seems to signify only html elements.

> Where a parameter is shown as accepting a node type, or a method returning a node type, it indicates that an object is expect / returned which refers to a DOM element.

Reply:
Minimally, the docs say that for here specifically,

> This may be a th or td element depending on the HTML for your table.

which means it is _always_ an HTMLElement and that `Node` is needlessly general.

Reply:
I see your point and agree. I made changes accordingly. Both `header()` and `footer()` functions return `HTMLElement` now. Thank your for the clarification.